src/Order.js

```diff
@@ -0,0 +1,17 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+export default function Order({ orderType }) {
+  const [order, setOrder] = useState(0);
+  function orderOne() {
+    setOrder(order + 1);
+  }
+  return (
+    <>
+      <li style={{ margin: "20px" }}>
+        {orderType}:{order}
+        <RestaurantButton orderOneProps={orderOne} />
+      </li>
+    </>
+  );
+}
```

Comment:
this component works well, by making a reusable component of `Order` you can apply the same logic & layout for multiple food items & each food item's count can be increased independently :)